OrbitGl/OrbitGlAccessibilityTest.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <functional>
+
+#include "AccessibilityInterfaceMock.h"
+
+namespace orbit_gl {
+
+TEST(AccessibilityRegistry, Management) {
+  TestA11yImpl* impl = new TestA11yImpl(nullptr);
```

Comment:
In this case, I chose to not use unique_ptr because I explicitely want to test the behavior on destruction (de-registration with `GlAccessibleInterfaceRegistry`), and I thought the explicit `delete` makes this more clear than just the unique_ptr going out of scope.

Reply:
Actually another and possibly better way to go would just be to use a scope. No need for dynamic memory allocation or `optional` at all.

Reply:
Although then you need to remember the old pointer so `unique_ptr` is probably just more convenient.

Reply:
I concede :D I have to agree that the `unique_ptr` and `.reset()` solution looks nicer and is as explicit as using the delete, changed it to `unique_ptr`.